lib/modules/platform/github/readme.md

```diff
@@ -2,8 +2,10 @@
 
 ## Authentication
 
-First, [create a classic Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic) for the bot account, select `repo` scope.
-Fine-grained Personal Access Tokens do not support the GitHub GraphQL API and cannot be used with Renovate.
+First, create a
+[fine-grained](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token)
+or [classic](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic)
+PAT ([Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#about-personal-access-tokens)) for the bot account, select `repo` scope.
```

Comment:
I think this should all be one line, as it's one sentence. But having it all on a single line is hard to read and maintain because of the long links. Would a short list work?

> First, create one of these:
> 
> - a fine-grained token,
> - or a classic token,
> - or a PAT

It looks like the user can choose between three options? The fine-grained token, a classic token, or a PAT? But the text right after proceeds to mention the PAT first.

So I think we need more changes to the docs than just inserting the fine-grained token stuff.

Reply:
PAT is a general category. Classic tokens and Fine-grained tokens are *both* PATs.

Reply:
Whoops, you're right! PAT is the _category_, and classic tokens and fine-grained tokens are a item in that category... 🙃 

In any case, I'd still like the sentence to be rewritten so it's easier to maintain.

Reply:
Doc writing isn't my day job. I do much more editing of other people's docs. I'm happy to apply correct changes and can sometimes iterate through things.

I'd be inclined to have a sentence (that mentions PATs) followed by two bullets because I think it's easier for a reader to parse, but I won't write it unless it'd be acceptable.

Reply:
```suggestion
First, create a [fine-grained](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token) _or_ a [classic](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic) PAT.
The PAT must have the `repo` scope.

Read the [GitHub Docs, about Personal Access Tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#about-personal-access-tokens) to learn more about PATs.
```

Let's go with this then. 😄 

I'm trying to split complicated stuff into their own sentences, so you don't have to remember too much in one go.